src/plugins/unified_search/public/query_string_input/text_based_languages_editor/resizable_button.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { EuiFlexGroup } from '@elastic/eui';
+
+import './resizable_button.scss';
+
+export function ResizableButton({
+  onMouseDownResizeHandler,
+  onKeyDownResizeHandler,
+}: {
+  onMouseDownResizeHandler: (
+    mouseDownEvent: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => void;
+  onKeyDownResizeHandler: (keyDownEvernt: React.KeyboardEvent) => void;
+}) {
+  const setFocus = (e: React.MouseEvent<HTMLButtonElement>) => e.currentTarget.focus();
+
+  return (
+    <div className="unifiedTextLangEditor--resizableButtonWrapper">
+      <EuiFlexGroup
+        direction="column"
+        gutterSize="none"
+        className="unifiedTextLangEditor--resizableButtonContainer"
+      >
```

Comment:
@MichaelMarcialis let me explain it. Maybe you can help with it. In order to move the button so that it appears on top of the border that divides the unified search UI and the rest of the page content I had to add an absolute flex div as a wrappen to the button. And because I need the absolute div I added one relative on the upper level. I can't think of a better solution here. If you have something better to suggest I am open to test it. I removed the FlexGroup and added the flex on the a div. Is it better?